packages/web/src/features/notice/componenets/NoticeList.tsx

```diff
@@ -0,0 +1,42 @@
+"use client";
+
+import React from "react";
+import styled from "styled-components";
+
+import NoticeListItem from "@sparcs-clubs/web/features/notice/componenets/NoticeListItem";
+import type { NoticeInfo } from "@sparcs-clubs/web/features/notice/types/notice.type";
+
+interface NoticeListProps {
+  infos: Array<NoticeInfo>;
+}
+
+const NoticeListInner = styled.div`
+  width: 100%;
+  flex-grow: 1;
+  flex-shrink: 1;
+  overflow: hidden;
+  border-bottom: 1px solid;
+  border-color: ${({ theme }) => theme.colors.GRAY[200]};
```

Comment:
사실 color도 그냥 한 속성에 같이 묶어서 작성해주면 좋을 것 같아요!! (다른 파일에서도 마찬가지로)

```suggestion
  border-bottom: 1px solid ${({ theme }) => theme.colors.GRAY[200]};
```